src/NexusMods.App/LoggingSettings.cs

```diff
@@ -40,6 +40,11 @@ public record LoggingSettings : ISettings
     /// </summary>
     public LogLevel MinimumLevel { get; init; } = LogLevel.Debug;
 
+    /// <summary>
+    /// When enabled, logs will be written to the console as well as the log file.
+    /// </summary>
+    public bool LogToConsole { get; init; } = false;
```

Comment:
Can we keep this on `true` for debug builds?

Reply:
Sure, I'll add that